ci/src/cI_github_hooks.mli

```diff
@@ -1,80 +1,17 @@
-open CI_utils
-open! Result
-
-type t
-
-module Commit_state : sig
-  type t
-
-  val status : t -> Datakit_S.status_state option Lwt.t
-  val descr : t -> string option Lwt.t
-  val target_url : t -> Uri.t option Lwt.t
-end
+open Datakit_github
 
 module CI : sig
-  type t
-  val of_string : string -> t
+  type t = string list
```

Comment:
Why is this no longer abstract? (doesn't matter much, but seems strange that we need to expose it now if we didn't before)